src/app/tags/tag-dialog/tag-dialog.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Component, Inject } from "@angular/core";
 import { FormBuilder, FormGroup } from "@angular/forms";
-import { MatDialogRef, MAT_DIALOG_DATA } from "@angular/material";
+import { MAT_DIALOG_DATA, MatDialogRef } from "@angular/material";
```

Comment:
if you change ![image](https://user-images.githubusercontent.com/14272396/38795539-4a1e48ea-4161-11e8-8ad5-96456a1da452.png) to ![image](https://user-images.githubusercontent.com/14272396/38795549-53ba6abe-4161-11e8-859f-180e672f8167.png), change it in all dialogs (in confirm dialog too"